### PR TITLE
Add specific shop frontend files

### PR DESCRIPTION
**Pull request by bavanya**

### CODE REVIEW

File: frontendservice/src/views/specificShop.vue [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+<template>
+  <div>
+    <NavBarCustomer></NavBarCustomer>
+    <cookieinfo></cookieinfo>
+    <cookieackno></cookieackno>
+    <div class="card">
+      <div class="card-content">
+        <p class="title">{{shopinfo.shop.shopName}}</p>
+        <p v-if="flag" class="title">Number of People ahead in the queue are:</p>
+        <p
```
Comment (KarishmaSinghChauhan) [inline frontendservice/src/views/specificShop.vue:18]: Nit. Code in lines 18 to 21 can be in the same line maybe.

File: frontendservice/src/views/specificShop.vue [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+          v-else
+          class="title"
+        >Number of people currently in queue are {{shopinfo.customersInQueue}}</p>
+        <p class="subtitle">{{shopinfo.shop.shopType}}</p>
+        <p class="subtitle">{{shopinfo.shop.address}}</p>
+        <p class="subtitle">{{shopinfo.shop.phoneNumber}}</p>
+      </div>
+      <footer class="card-footer">
+        <p class="card-footer-item">
+          <button
```
Comment (KarishmaSinghChauhan) [inline frontendservice/src/views/specificShop.vue:28]: Nit. Same for lines 28 to 32.

File: frontendservice/src/views/specificShop.vue [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+            class="button is-primary is-medium"
+            v-if="flag"
+            v-on:click="getShopInfo()"
+          >Refresh token</button>
+          <button class="button is-primary is-medium" v-else v-on:click="getToken()">Take token</button>
+        </p>
+        <p class="card-footer-item">
+          <button
```
Comment (KarishmaSinghChauhan) [inline frontendservice/src/views/specificShop.vue:36]: Same for lines 36 to 40.

File: frontendservice/src/views/specificShop.vue (hunk 2 above)
Comment (bavanya) [reply frontendservice/src/views/specificShop.vue:28]: Same here happening automatically when I save the code in vscode

File: frontendservice/src/views/specificShop.vue (hunk 1 above)
Comment (bavanya) [reply frontendservice/src/views/specificShop.vue:18]: Same here happening automatically when I save the code in vscode

File: frontendservice/src/views/specificShop.vue (hunk 3 above)
Comment (bavanya) [reply frontendservice/src/views/specificShop.vue:36]: Same here happening automatically when I save the code in vscode

**Pull request merged**